src/main/java/com/minecolonies/coremod/colony/buildings/BuildingCook.java

```diff
@@ -187,7 +189,7 @@ public View(final ColonyView c, final BlockPos l)
         @Override
         public Window getWindow()
         {
-            return new WindowHutWorkerPlaceholder<>(this, COOK);
+            return new WindowHutWorkerPlaceholder<>(this, COOK_JOBNAME);
```

Comment:
Is this user facing? If yes, translation!

Reply:
Nope, not user facing, this here is for registry mainly